router/batchrouter/asyncdestinationmanager/bing-ads/util.go

```diff
@@ -126,7 +126,7 @@ func (b *BingAdsBulkUploader) populateZipFile(actionFile *ActionFileInfo, audien
 		for _, uploadData := range data.Message.List {
 			clientIdI := newClientID(data.Metadata.JobID, uploadData.HashedEmail)
 			clientIdStr := clientIdI.ToString()
-			actionFile.CSVWriter.Write([]string{"Customer List Item", "", "", audienceId, clientIdStr, "", "", "", "", "", "", "Email", uploadData.Email})
+			actionFile.CSVWriter.Write([]string{"Customer List Item", "", "", audienceId, clientIdStr, "", "", "", "", "", "", "Email", uploadData.HashedEmail})
```

Comment:
why HashedEmail we are sending here? Because the transformer code works like below:
There are two keys, Email and Hashed email. Now hashed email will always have hashed value, even if the user has opted for non hashing the emails from UI. 
The hashing is being controlled for only the email key based on user's choice. 
If we put hashed email here, and the user is sending an already hashed data, we are basically sending double hashed data via server, not the actual data user sent.
That is why I edited it to uploadDtata.Email and gave the transformer the liberty to send data based on user configuration WDYT @koladilip @ItsSudip

Reply:
The hashedEmail is helping us to create clientId